### DISCUSSION TITLE
Table - v2 vs. v3

### DISCUSSION THREAD

**Discussion by ovaisa** [iTwin/itwinjs-core#3874] [General]
I am converting a Table I had implemented in v2 to the v3 (iTwinUi) Table.  Can you please guide where I can find out info on the parameters required to set for the iTwinUi Table.  I am using the following link

https://itwin.github.io/iTwinUI-react/?path=/docs/core-table--filters   ->> Docs

It defines several parameters (className, column, data, density ...), I am unable to find exactly what I need to pass especially for the 'column' parameter (I assume this is same as the 'column' in v2 but without documentation I can't know for sure). Similarly I want to use 'selectionMode' and perhaps 'style', not sure what to pass.  v2 had very extensive documentation on these parameter but I am unable to locate documentation for v3.  Can you please guide where I can find documentation and details on these parameters.

In v2, I was using the 'showDistinctValueFilters' option

![TableFilter](https://user-images.githubusercontent.com/83007711/176147631-94e76041-f7e0-4e78-8843-f049915c34b6.JPG)

In iTwinUi table, I only get option for filter

![image](https://user-images.githubusercontent.com/83007711/176148864-0f589b11-b136-41b2-9ba4-b8622fab8024.png)

Can you please guide how I can enable the 'showDistinctValueFilters'

Thank you,

Ovais

**Comment by ovaisa** [comment]
Hi, any update on this issue.

**Reply by pmconne** [reply]
@NancyMcCallB

**Reply by aruniverse** [reply]
@veekeys @bentleyvk @mayank99

**Reply by mayank99** [reply]
Hi @ovaisa, have you checked the code snippets that come with the iTwinUI Table examples? It's in the Canvas tab in Storybook.

![ ](https://user-images.githubusercontent.com/9084735/177569709-cb420849-5e95-4e5f-9681-a4a95b669fb8.png)

Also worth mentioning that we wrap [`react-table-v7`](https://react-table-v7.tanstack.com/docs/overview) so you can look at their website for detailed docs.

**Reply by hanan-commits-work** [reply]
Greetings.
The "dropdown selection" in the filter was available in 2.0 out of the box and the developers didnt need to implement this capability from the scratch. we couldnt find this feature in 3.0, so I assume it is removed, but since it is deprecated in 3.0, we have to reinvent the wheel just for this basic functionality that was available in the previous version. 
Can you please let us know if this capability is being developed or we just have to completely shift over to React-Table.

**Reply by bentleyvk** [reply]
Hey, if you feel that you are missing something in the Table feel free to [create us an issue](https://github.com/iTwin/iTwinUI-react/issues/new/choose). For now, you can check this this discussion https://github.com/iTwin/iTwinUI-react/discussions/691, here one of our lib user created codesandbox with various custom made filters.
